ethcore/src/miner/miner.rs

```diff
@@ -764,9 +764,11 @@ impl MinerService for Miner {
 			if let Some(ref ap) = self.accounts {
 				ap.sign(address.clone(), Some(password.clone()), Default::default())?;
 			}
-			let mut sealing_work = self.sealing_work.lock();
-			sealing_work.enabled = self.engine.is_sealer(&address).unwrap_or(false);
-			*self.author.write() = address;
+			{
```

Comment:
good to have a comment here to ensure any newcomer understands what the parens are for.